source/manual/alerts/data-sync.html.md

```diff
@@ -25,6 +25,8 @@ Check the output of the production Jenkins job to see which part of the sync fai
 > that rely on GDS API Adapters are likely to fail due to authentication
 > failures, as well as Smokey tests that attempt to use Signon.
 
+A failure may mean that links to apps in Signon in staging link to production apps. Running the [data_sync_complete](https://deploy.staging.publishing.service.gov.uk/job/Data_Sync_Complete/) job may fix this.
```

Comment:
This line seems to contain the same information as the above warning.

If you do want to make a change here, it looks like it would be worth consolidating the information.

Reply:
Excellent point, hows this?

Reply:
That looks fine now :+1: